proposed/form.md

```diff
@@ -0,0 +1,101 @@
+# Parsy Pattern Design Doc
+
+Parsy
+
+#Form
+
+* Author: `Lorenzo Wallerlei, lorenzo@polderknowledge.nl`
+* Date: `19-01-2017`
+* Repository name: `FormComponent`
+
+## Goals
+
+This document is the proposal for the form component, it describes what should go in a form and how a form should react 
+to user input.
+
+## References
+
+http://getbootstrap.com/components/#input-groups
+
+http://foundation.zurb.com/sites/docs/forms.html
+
+http://webaim.org/techniques/forms/controls
+
+## Requirements
+
+- All the inputs within the form SHOULD contain a tab index.
+- All the inputs SHOULD have error messages if there is some sort of validation on that specific input
+- Inputs MUST have labels
+- Form MUST have an action attribute
+- If there is a collection of inputs it MUST be surrounded by a fieldset.
+- If you use an fieldset you MUST use a legend for it.
+
+# Case studies `<Optional>`
+
+Not applied.
+
+## Existing implementations
+
+[https://www.tax.service.gov.uk/gg/sign-in?continue=%2Faccount&origin=unknown]
+
+What it does good.
+- Form fields are described by labels.
+- Flow of the tab indexes is natural.
```

Comment:
The login form at this URL does not have tabindex attributes set. The flow is logical because by itself already. Just saying :)